src/Games/NexusMods.Games.MountAndBlade2Bannerlord/Utils/GamePathProvier.cs

```diff
@@ -0,0 +1,30 @@
+using Bannerlord.LauncherManager;
+using NexusMods.Abstractions.GameLocators;
+using NexusMods.Paths.Extensions;
+using GameStore = NexusMods.Abstractions.GameLocators.GameStore;
+
+namespace NexusMods.Games.MountAndBlade2Bannerlord.Utils;
+
+public static class GamePathProvier
+{
+    private static string GetConfiguration(GameStore store) =>
+        LauncherManagerHandler.GetConfigurationByPlatform(LauncherManagerHandler.FromStore(Converter.ToGameStoreTW(store)));
+
+    public static GamePath PrimaryLauncherFile(GameStore store) =>
```

Comment:
The game entry in `MountAndBlade2Bannerlord.cs` calls into this.

Although Xbox store isn't yet fully supported, I imagine you'd want to switch over `store` variable here and change the file name for Xbox store. Because the current logic  would use `TaleWorlds.MountAndBlade.Launcher.exe` when `Launcher.Native.exe` is likely intended when store is Xbox.

Don't have the title on Xbox available to me right now, but I believe this to likely be the case.

(Note: Xbox store basically works but App fails due to missing read privileges on binaries. I know how to fix that, I do it in one of my own projects; so I'll eventually get to pick it up some day 🤞)

Reply:
In our case it's simpler. We don't use the original executable on Xbox.
Xbox is only supported with BLSE, on other sores we support both the Vanilla executable and BLSE

Reply:
Does Xbox not have a native module manager?

Would we need to eventually emit a diagnostic on Xbox one day to tell the user to download+install BLSE?

Reply:
Xbox uses the same module management, but the way the game ships kinda broke out main mods.
I don't remember the exact details, I believe it was because we need a controllable entrypoint to fix a critical Harmony issue (the game ships an older Mono.Cecil library that was eagerly loaded by the game's .NET hosting bootstrap). The issue breaks Harmony 2.3.x because we expect a higher version of that library.

The custom executable fixes the issue, but because of that we lose "proper" Xbox support like cloud saves. The game basically falls back to Steam/GOG/Epic behavior and uses save files from there.

For now yea, we need a diagnostic for that. Vortex just ask the user directly to download and install BLSE automatically (with a button confirmation of the action)

Reply:
Hmm, I see; that makes quite a bit of sense.
In this case, steering those users towards installing does make sense, yup.